modules/oxide/src/less/theme/components/view/view.less

```diff
@@ -64,7 +64,7 @@
     gap: @view-toolbar-button-spacing-x; // spacing between buttons within a group
     justify-content: space-between;
     overflow-x: auto;           // Show horizontal scrollbar on small screens
-    padding: @view-header-padding;
+    padding: 10px 10px 2px 10px;
```

Comment:
Why the hardcoded value shouldn't you change the variable here?
What effects does this have on other things that use this like the inline code view?

Reply:
My intention was to limit this change to what revision history is using but it is not consistent when the toolbar has no button groups.  To keep the consistency, I changed the variable instead. It would affect toolbar's padding with/without button groups.